pom.xml

```diff
@@ -244,11 +244,11 @@
 				</exclusion>
 			</exclusions>
 		</dependency>
-		<dependency>
+		<!-- <dependency>
 			<groupId>javax.xml.bind</groupId>
 			<artifactId>jaxb-api</artifactId>
 			<version>2.4.0-b180830.0359</version>
-		</dependency>
+		</dependency> -->
```

Comment:
I only see the two of these, so commenting both out might have unintended consequences?